app/views/foreman_rh_cloud/react/inventory_upload.html.erb

```diff
@@ -0,0 +1,3 @@
+
+<%= render "application" %>
```

Comment:
Why do you need it here? If this line is absolutely necessary, you can move it to layout I suppose.